.github/workflows/bash_unit_testing.yml

```diff
@@ -1,7 +1,6 @@
 name: bash Unit Testing
 
 on:
-  push:
```

Comment:
Does removing this mean the action only runs once when the pull request is created?

Reply:
Yes.  As it stands this runs whenever there's a commit, so for example when you push a branch without creating a PR or more commonly when a PR is merged then it runs against master, neither of which is helpful or done by any other of our checks.

Reply:
What about when a new commit is pushed to this same branch/PR?

Reply:
The line below catches that and is all any of our other actions use./